src/main/python/rlbot/utils/structures/game_interface.py

```diff
@@ -277,16 +277,11 @@ def get_live_data_flat_binary(self):
             self.game_status(None, RLBotCoreStatus.Success)
             return proto_string
 
-    def get_physics_tick(self) -> PhysicsTick:
+    def update_rigid_body_tick(self, rigid_body_tick: RigidBodyTick):
```

Comment:
Do you think it's worth keeping both implementations here for testing purposes? (maybe suffix the FlatBuffer one with `_flatbuffer`)

Reply:
I'd rather not, unless we identify a production use case. Everything we add to python goes into a gray area where it's a quasi-public API. It's not necessarily documented as a public API, but there's nothing stopping people from finding it and using it, so it becomes uncomfortable to change. I'd like to keep that surface area small if possible.